release-please-config.json

```diff
@@ -74,6 +74,9 @@
     "packages/calcite-design-tokens": {
       "component": "@esri/calcite-design-tokens"
     },
+    "packages/calcite-tailwind-preset": {
+      "component": "@esri/calcite-tailwind-preset"
```

Comment:
After setting the version to `0.0.0` in `package.json`, I'm pretty sure it will version to `1.0.0`, but it might be `0.1.0`. To be sure, we can add a `release-as` field to hardcode v1. We will need to remove the field after the release.

https://github.com/googleapis/release-please/blob/422df5a6a9d44a52c6c682563d1795b8fd246215/docs/manifest-releaser.md?plain=1#L296-L298